sql/core/src/test/resources/test-data/xml-resources/root-level-value.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<ROWSET>
+    <ROW>value1</ROW>
+    <ROW attr="attr1">value2</ROW>
+    <ROW>
+        value3
+        <!-- this is a comment-->
+    </ROW>
+</ROWSET>
```

Comment:
If possible, I would prefer put this XML into a string variable within `XmlSuite.scala` to make the test easier to read, and self-contained.

Reply:
Sure. I can do it in a follow-up PR.